aten/src/ATen/native/native_functions.yaml

```diff
@@ -4437,7 +4437,7 @@
   use_c10_dispatcher: full
   variants: function
 
-- func: choose_qparams_optimized(Tensor input, int numel, int n_bins, float ratio, int bit_width) -> (float, float)
+- func: choose_qparams_optimized(Tensor input, int numel, int n_bins, float ratio, int bit_width) -> (Tensor, Tensor)
```

Comment:
I was wondering the reason for having numel as an argument? Isn't it the same as input tensor size?

Reply:
Not necessarily, it could be per-channel in which case it will be numel per channel